pin/pin.go

```diff
@@ -400,7 +400,9 @@ func (p *pinner) isPinnedWithType(c cid.Cid, mode Mode) (string, bool, error) {
 		}
 		has, err := hasChild(p.internal, rc, c, visitedSet.Visit)
 		if err != nil {
```

Comment:
One quick question.. do you remember what error message did you get with ipld.GetLinks()?
Because this method is used in other places.

Reply:
It's the `merkledag: not found` error